Source/ACE.DatLoader/DatReader.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.IO;
+
+namespace ACE.DatLoader
+{
+    public class DatReader
+    {
+        public int Offset { get; set; }
+        public byte[] buffer { get; private set; }

```

Comment:
Thanks! It was private previously, but I exposed the getter so I could still export the content to a file using the console "portal-export" command.